src/api/Base.service.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable */

```

Comment:
could you please clarify what is the reason of disabling eslint?

Reply:
@DimatiBart, it came from @GlebShaucou MR. We already discussed it there.

Reply:
@AfanasyShevtsov could you clarify it then? 🤔or drop a link?